service/history/workflow/activity.go

```diff
@@ -76,3 +88,43 @@ func updateActivityInfoForRetries(
 
 	return ai
 }
+
+func NextBackoffInterval(
```

Comment:
can't find usages of this either.

Reply:
yeah, it was an attempt to refactor a bit. Will remove it